src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -39,42 +42,43 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
 
       val actual = outputs.map { case (k, WdlString(string)) => k -> string }
       actual shouldEqual Map(HelloWorld.OutputKey -> HelloWorld.OutputValue)
+      Await.result(dataAccess.shutdown(), Duration.Inf)
```

Comment:
:+1: for loan pattern

aside: Beyond thinking [this talk](https://www.parleys.com/tutorial/unsung-heroes-less-fashionable-patterns-scala) from Dick Wall is worthwhile he does have an amusing anecdote about bioinformaticists abusing the loan pattern (just after 13:00 if you want to skip the rest)